.github/actions/tests/slack-alert-on-pr-merge/scripts/compose-slack-message.js

```diff
@@ -0,0 +1,114 @@
+module.exports = () => {
+	const {
+		GITHUB_BASE_REF,
+		GITHUB_RUN_ID,
+		PR_NUMBER,
+		PR_TITLE,
+		SHA,
+		TEST_TYPE,
+	} = process.env;
+
+	// Slack message blocks
+	const blocks = [];
+	const dividerBlock = {
+		type: 'divider',
+	};
+	const introBlock = {
+		type: 'section',
+		text: {
+			type: 'mrkdwn',
+			text: `${ TEST_TYPE } tests failed on \`${ GITHUB_BASE_REF }\` after merging PR <https://github.com/woocommerce/woocommerce/pull/${ PR_NUMBER }|#${ PR_NUMBER }>`,
+		},
+	};
+	const prTitleBlock = {
+		type: 'header',
+		text: {
+			type: 'plain_text',
+			text: PR_TITLE,
+			emoji: true,
+		},
+	};
+	const prButtonBlock = {
+		type: 'actions',
+		elements: [
+			{
+				type: 'button',
```

Comment:
Not a big deal or a blocker but by using buttons here instead of markdown I get warning after clicking them `This app is not configured to handle interactive responses.`

The buttons still work so not a blocker.

![Screenshot 2023-03-10 at 21 02 24](https://user-images.githubusercontent.com/24649833/224427921-884fe317-5155-425b-bd6e-9bd792469896.png)
![Screenshot 2023-03-10 at 21 04 15](https://user-images.githubusercontent.com/24649833/224428103-e9eba5e1-1011-4793-a5cb-e7fbf50c87bf.png)

Reply:
Yeah, I do see those warnings too. I created a follow-up task to address this: https://github.com/woocommerce/woocommerce/issues/37187